src/app/MessageProvider.tsx

```diff
@@ -0,0 +1,22 @@
+
+import React from 'react';
+import { addLocaleData, IntlProvider } from 'react-intl';
+import en from 'react-intl/locale-data/en';
+
+addLocaleData(en);
+
+const messages = {
+  'i18n:404': 'page not found',
+  'i18n:a11y:skipToContent': 'Skip to Content',
+  'i18n:toc:bookDetails': 'book details',
+  'i18n:toc:title': 'Table of Contents',
+  'i18n:toc:toggle:closed': 'Click to open the Table of Contents',
+  'i18n:toc:toggle:opened': 'Click to close the Table of Contents',
```

Comment:
is there some value to the `i18n:` prefix on everything?

Reply:
I added the prefix initially to distinguish the i18n strings from other strings in the code like `opened` and to easily search-and-replace later. I figured it's easier to remove later than to add later. 

Can we keep it until we've explored ways for CI to verify that all strings are i18n-ized ? I'm not sure how to do that but one way is to just not add the lookup table in tests... then the tests could verify that `body.innerHTML` only contains strings that begin with `i18n:` or something.

Reply:
yea we can leave it for now